packages/core/services/AnnotationService/DatabaseAnnotationService/index.ts

```diff
@@ -109,7 +109,9 @@ export default class DatabaseAnnotationService implements AnnotationService {
         const rows = await this.databaseService.query(sqlBuilder.toSQL());
         const rowsSplitByDelimiter = rows
             .flatMap((row) =>
-                isNil(row[annotation]) ? [] : row[annotation].split(DatabaseService.LIST_DELIMITER)
+                isNil(row[annotation])
+                    ? []
+                    : `${row[annotation]}`.split(DatabaseService.LIST_DELIMITER)
```

Comment:
is this safe from annotations that contain ','?  eg, Baseball Teams: ["tom,dick,harry","larry,curly,moe"]

Reply:
Nope! The hope is to provide an option to choose the delimeter in the future